src/rgw/rgw_bucket_layout.h

```diff
@@ -170,6 +173,10 @@ struct bucket_log_layout {
 
   bucket_index_log_layout in_index;
 
+  BucketLogType type2 = BucketLogType::Deleted;
```

Comment:
@cbodley should we have a separate struct for the new type? I'm not even using type2 but I'm required to have the bucket_index_log_layout for the new type here to support encoding/decoding the BucketLogType

Reply:
yup makes sense. thanks. done!